pom.xml

```diff
@@ -222,7 +222,7 @@
     <maven.versions.plugin.version>2.13.0</maven.versions.plugin.version>
     <maven.compiler.plugin.version>3.10.1</maven.compiler.plugin.version>
     <java.version>11</java.version>
-    <pitest.plugin.version>1.9.9</pitest.plugin.version>
+    <pitest.plugin.version>1.11.0</pitest.plugin.version>
```

Comment:
@romani It looks like something is wrong with dependabot as it has stopped updating this plugin.

Last update https://github.com/checkstyle/checkstyle/pull/12293

https://github.com/hcoles/pitest/releases
We are missing plugins 1.9.10 through 1.10.4

Reply:
We can send them feedback to investigate and improve.

We have azure weekend job to remind us on any dependency that dependabot does not see.

Reply:
This PR should help a lot in update of suppressions

Reply:
@romani 
> We have azure weekend job to remind us on any dependency that dependabot does not see.

Can you look to see if it has been reporting pitest? I cleaned up a bunch of my emails and may have removed it.

Last update by dependa was in October. So if azure if reporting, it should have been reporting for 3-4 months.

Reply:
I do when I have time, but definetely not case in last several month

Reply:
You provided me the details in PM. New issue clarified why it has not been reporting.